src/os/osexec.go

```diff
@@ -0,0 +1,56 @@
+//go:build linux
+// +build linux
+
+package os
+
+import (
+	"errors"
+	"syscall"
+	"unsafe"
+)
+
+func Fork() (pid int, err error) {
```

Comment:
Please unexport `Fork` and `Execve` because they aren't provided by the "big Go" os package (and are easy to misuse).